tests/driver_bmp180/Readme.md

```diff
@@ -0,0 +1,15 @@
+## About
+This is a test application for the BMP180 Pressure and Temperature sensor.
+
+## Usage
+The application will initialize the BMP180 sensor and display its calibration
+coefficients. Please see section 3.4 of the
+[datasheet](https://www.adafruit.com/datasheets/BST-BMP180-DS000-09.pdf) for
+more information.
+
+After initialization, every 2 seconds, the application:
+* reads temperature (°C);
+* reads pressure (hPa) ; and prints them to the STDOUT;
+* computes the pressure at sea level (assuming the current pressure is computed
+based on Polytechnique School Campus altitude, 158m);
+* computes the altitude (given the pressure at sea level).
```

Comment:
`given the pressure at sea level` => `based on the pressure at sea level calculated above`